.gitignore

```diff
@@ -4,6 +4,8 @@ syntax: glob
 
 # Tool Runtime Dir
 /[Tt]ools/
+.dotnet/
+artifacts
```

Comment:
Also `.packages/` maybe, if you follow the Arcade convention.

Reply:
Thanks, added.